Source/DSPTool/DSPTool.cpp

```diff
@@ -95,21 +102,22 @@ static void CodesToHeader(const std::vector<u16>* codes, const std::vector<std::
   header.append("};\n\n");
   header.append("const unsigned short dsp_code[NUM_UCODES][0x1000] = {\n");
 
-  for (u32 i = 0; i < num_codes; i++)
+  for (std::size_t i = 0; i < codes.size(); i++)
```

Comment:
This might be a candidate for a range-based for loop as well; but not the inner loop I guess...

Reply:
It wouldn't because `codes_padded[i]` is used in the inner loop expression.

Reply:
Ugh, missed that one :/